doc/github_actions.rst

```diff
@@ -0,0 +1,35 @@
+Integration with GitHub Actions
+===============================
+
+`Universum` requires no special integration with `GitHub Actions <https://docs.github.com/en/actions>`_. It is usually
+launched as one long step in a single build stage.
+
+.. warning::
+
+    GitHub Actions doesn't support `report_artifacts` key in configuration.
+
+Command line
+------------
+
+To use `universum` to check-out current repository (the repository that contains the GitHub Action workflow) is
+necessary to use `GitHub Environment variables
+<https://docs.github.com/en/actions/learn-github-actions/environment-variables>`_:
+``python -m universum --vcs-type=git --git-repo $GITHUB_SERVER_URL/$GITHUB_REPOSITORY --git-refspec $GITHUB_REF_NAME``
```

Comment:
Is "$GITHUB_SERVER_URL/$GITHUB_REPOSITORY" a path or a choice?

Reply:
path

Reply:
Then please explain that in the instruction, please.

Reply:
how ? please show an example

Reply:
No examples, just describe it with your own words.